doc/man3/flux_request_decode.adoc

```diff
@@ -22,7 +22,7 @@ SYNOPSIS
 
  int flux_request_decode_raw (const flux_msg_t *msg,
                               const char **topic,
-                              void *data, int *len);
+                              void **data, int *len);
```

Comment:
For me, when I first looked at:

```
int flux_request_decode_raw (const flux_msg_t *msg, const char **topic, void *data, int *len);
```

and compared it to

```
flux_msg_t *flux_request_encode_raw (const char *topic, const void *data, int len);
```

My thought was "how come topic & len are in & out parameters, but not data?"  You would think it should be a ```void **```.